docker/pg_tests/scripts/tests/delete_before_permanent_test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+set -e -x
+
+export WALE_S3_PREFIX=s3://deletebeforepermanentbucket
+export WALG_USE_WAL_DELTA=true
+export WALG_DELTA_MAX_STEPS=0
+
+/usr/lib/postgresql/10/bin/initdb ${PGDATA}
+
+echo "archive_mode = on" >> /var/lib/postgresql/10/main/postgresql.conf
+echo "archive_command = '/usr/bin/timeout 600 /usr/bin/wal-g wal-push %p'" >> /var/lib/postgresql/10/main/postgresql.conf
+echo "archive_timeout = 600" >> /var/lib/postgresql/10/main/postgresql.conf
+
+/usr/lib/postgresql/10/bin/pg_ctl -D ${PGDATA} -w start
+
+# push first backup as permanent
```

Comment:
This way you check only full backup permanence. Add another test scenario that will check delta backup permanence.
Let it be something like:
1. Make several non-permanent backups.
2. Make permanent delta backup.
3. Add some data to DB.
4. Make another non-permanent delta backup.
5. Run 'delete' that will delete the latest backup.
6. Restore LATEST backup and check that DB has the same state that it had before permanent backup.